test-network/organizations/fabric-ca/reEnroll.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+function reEnrollOrg1() {
+  infoln "Re-enrolling the CA admin"
+  mkdir -p organizations/peerOrganizations/org1.example.com/
+
+  export FABRIC_CA_CLIENT_HOME=${PWD}/organizations/peerOrganizations/org1.example.com/
+
+  set -x
+  fabric-ca-client reenroll -u https://admin:adminpw@localhost:7054 --caname ca-org1 --tls.certfiles "${PWD}/organizations/fabric-ca/org1/ca-cert.pem" --csr.keyrequest.reusekey
+  { set +x; } 2>/dev/null
+
+  # Copy org1's CA cert to org1's /msp/tlscacerts directory (for use in the channel MSP definition)
+  mkdir -p "${PWD}/organizations/peerOrganizations/org1.example.com/msp/tlscacerts"
+  cp "${PWD}/organizations/fabric-ca/org1/ca-cert.pem" "${PWD}/organizations/peerOrganizations/org1.example.com/msp/tlscacerts/ca.crt"
+
+  # Copy org1's CA cert to org1's /tlsca directory (for use by clients)
+  mkdir -p "${PWD}/organizations/peerOrganizations/org1.example.com/tlsca"
+  cp "${PWD}/organizations/fabric-ca/org1/ca-cert.pem" "${PWD}/organizations/peerOrganizations/org1.example.com/tlsca/tlsca.org1.example.com-cert.pem"
+
+  # Copy org1's CA cert to org1's /ca directory (for use by clients)
+  mkdir -p "${PWD}/organizations/peerOrganizations/org1.example.com/ca"
+  cp "${PWD}/organizations/fabric-ca/org1/ca-cert.pem" "${PWD}/organizations/peerOrganizations/org1.example.com/ca/ca.org1.example.com-cert.pem"
+
+
+  infoln "Generating the peer0 msp"
+  set -x
+  fabric-ca-client reenroll -u https://peer0:peer0pw@localhost:7054 --caname ca-org1 -M "${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/msp" --csr.hosts peer0.org1.example.com --tls.certfiles "${PWD}/organizations/fabric-ca/org1/ca-cert.pem" --csr.keyrequest.reusekey
+  { set +x; } 2>/dev/null
+
+  cp "${PWD}/organizations/peerOrganizations/org1.example.com/msp/config.yaml" "${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/msp/config.yaml"
```

Comment:
I don't think anything in the config.yaml changes when you reenroll. Did you find something that changed?